tests/rptest/tests/tx_admin_api_test.py

```diff
@@ -16,9 +16,12 @@
 
 
 class TxAdminTest(RedpandaTest):
-    topics = (TopicSpec(name="tx_test",
+    topics = (TopicSpec(name="tx_test1",
                         partition_count=3,
-                        replication_factor=3), )
+                        replication_factor=3),
+              TopicSpec(name="tx_test2",
```

Comment:
can you drop the `name` and let `TopicSpec` assign a random topic name?

Reply:
Why I can not use my name for topics?

Reply:
You can use your own topic names, but allowing the topic name to be generated for you results in tests that are more generic. That is potentially important in the future for how we run tests against a cluster. For example, how do you know another test didn't choose a topic with the same name or if multiple instances of this topic are running in parallel.

Feel free to use a custom name if it is important to the test, but I think that wasm has so far been the only case where this is needed.